source/parameters/meta.md

```diff
@@ -2,17 +2,32 @@
 (s:meta.params)=
 ## MetaP: Metadata Parameters
 
-The `MetaP` parameter group can be used for metadata that describes the lattice element
-but is not part of the PALS standard. Such data is necessarily program dependent.
-Example metadata could be blueprint information, information on power supply connections, etc.
-Example:
+`MetaP` has four standard components
 ```{code} yaml
-MetaP:
-  alias:  an_alternative_name
-  Meta: Mark 4 quadrupole
+  alias         # String
+  ID            # String
+  label         # String
+  description   # String
 ```
+In addition to an element's name, these strings can be used for pattern matching
+when trying to locate all elements of a given type.
+
+Besides the four standard strings, the `MetaP` parameter group can be used to store metadata 
+that describes the lattice element but is not part of the PALS standard. 
+Such data is necessarily program dependent.
+Example metadata could be blueprint information, information on power supply connections, etc.
 
 Components of `MetaP` are not limited to being simple strings or numbers but can be complex 
 structures. The information contained in `MetaP` should be restricted to information that 
 does not affect simulations. Custom, program specific information should be stored elsewhere.
+Example:
+```{code} yaml
+Metap:
+  ID: 0137-85
+  history:                  # Custom information
```

Comment:
Thank you for adding the example!

I am a bit confused that only `ID` of the four components mentioned above is used.

We need to be specific what the four components above are used for, if they are required, and if they are optional, then if their default value is well-defined or undefined.

Reply:
The standard now states that all parameters are optional unless otherwise stated and strings default is, by default, blank.